eng/config/globalconfigs/ApiShim.globalconfig

```diff
@@ -1,10 +1,15 @@
-is_global = true
-
 # API-related warnings do not make sense in API shim files
 
+[*.cs]
 # CA1816: Call GC.SuppressFinalize correctly
 dotnet_diagnostic.CA1816.severity = none
 
+# CA1822: Member 'Member' does not access instance data and can be marked as static
+dotnet_diagnostic.CA1822.severity = none
+
+# CA2227: Change 'Member' to be read-only by removing the property setter
+dotnet_diagnostic.CA2227.severity = none
```

Comment:
These are not needed in this file, since they are already defined in NonShipping.globalconfig (which has wider scope than this file).

Reply:
Yes but they're a suggestion in `NonShipping.globalconfig`, which for the APIShims is strictly wrong (since acting on that suggestion would break our tests pretty hard). So if someone were to come through and run "Fix all for solution" on a different instance of CA2227 now things are broken.